src/components/MoneyRequestHeader.tsx

```diff
@@ -81,12 +81,6 @@ function MoneyRequestHeader({session, parentReport, report, parentReportAction,
 
     const isRequestModifiable = !isSettled && !isApproved && !ReportActionsUtils.isDeletedAction(parentReportAction);
     const canModifyRequest = isActionOwner && !isSettled && !isApproved && !ReportActionsUtils.isDeletedAction(parentReportAction);
-    let canDeleteRequest = canModifyRequest;
-
-    if (ReportUtils.isPaidGroupPolicyExpenseReport(moneyRequestReport)) {
-        // If it's a paid policy expense report, only allow deleting the request if it's not submitted or the user is the policy admin
-        canDeleteRequest = canDeleteRequest && (ReportUtils.isDraftExpenseReport(moneyRequestReport) || PolicyUtils.isPolicyAdmin(policy));
```

Comment:
This is the change required for allowing submitted reports to be edited. The `isPolicyAdmin` check does nothing here actually, as it conflicts with `canDeleteRequest` which requires the user to be the owner of the report.